public/index.html

```diff
@@ -259,40 +274,41 @@ <h3 class="solution__section__hdr">She finds you deals.</h3>
-        <img class="solution__section__img6" src="assets/images/section6-illo.svg" alt="Florence holding a GO sign">
+        <img class="solution__section__img6-bg solution__section__bg" src="assets/images/section6-illo-bg.jpg" alt="background paint splash">
+        <img class="solution__section__img6 solution__section__img" src="assets/images/section6-illo.svg" alt="Florence holding a GO sign">
       </div>
       <div class="solution__section__right">
         <figure>
           <p class="screen-reader">An example text from Florence to a patient</p>
           <ol class="bubble">
             <li class="bubble__florence">Hey Mike! Nice job, your cholesterol is down 20%.</li>
-            <li class="bubble__florence--tail">Treat yourself to something nice! <img class="emoji" src="assets/images/happy.svg" alt="smile emoji"></li>
+            <li class="bubble__florence bubble__florence--tail">Treat yourself to something nice! <img class="emoji" src="assets/images/smile.svg" alt="smile emoji"></li>
           </ol>
         </figure>
-        <img class="solution__section__right__avatar--6" src="assets/images/florence-avatar.svg" alt="Florence avatar">
+        <img class="avatar avatar--6" src="assets/images/florence-avatar.svg" alt="Florence avatar">
       </div>
     </section>
 
     <section class="solution__section--7">
       <div class="solution__section__left">
         <h3 class="solution__section__hdr">She keeps you honest.</h3>
         <p class="solution__section__content">If Florence notices you’re not staying on top of your plan, she can message family members or others in your support network.</p>
-        <img class="solution__section__img7-bg" src="assets/images/section7-illo-bg.jpg" alt="background paint splash">
-        <img class="solution__section__img7" src="assets/images/section7-illo.svg" alt="telephone handset">
+        <img class="solution__section__img7-bg solution__section__bg" src="assets/images/section7-illo-bg.jpg" alt="background paint splash">
+        <img class="solution__section__img7 solution__section__img" src="assets/images/section7-illo.svg" alt="telephone handset">
       </div>
       <div class="solution__section__right">
         <figure>
           <p class="screen-reader">An example text conversation from Florence to a patient's sister</p>
           <ol class="bubble">
-            <li class="bubble__florence--tail">Hi Kim, this is an automated message from FlorenceApp.io Your brother Mike hasn't taken his meds in three days. You might want to give him a call. <a class="bubble__number" href="">(919) 232-0232</a></li>
+            <li class="bubble__florence bubble__florence--tail">Hi Kim, this is an automated message from FlorenceApp.io Your brother Mike hasn't taken his meds in three days. You might want to give him a call. <a class="bubble__number" href="">(919) 232-0232</a></li>
```

Comment:
A trick with phone number links:

```
<a href="tel:1-919-232-0232">(919) 232-0232</a>
```

If you're viewing this site on a phone, then you can now tap that number to call 😄 (Plus some folks that have phone apps on their desktop computer.)

Reply:
oh sweeet!!! thanks!
resolved in pr#4